litmus-portal/Makefile

```diff
@@ -76,6 +76,16 @@ lint-checks:
 		echo "gofmt checking failed!"; echo "$${fmtRes}"; echo; \
 		echo "Please ensure you are using $$($(GO) version) for formatting code."; \
 	fi
+	@echo "------------------"
```

Comment:
Is this change ok? @rajdas98

Reply:
@rajdas98  Please review this. It's been long time I am busy with this PR.
@arkajyotiMukherjee  @gdsoumya  I made require updates as we discuss in the meet. But for now Workflow Cluster and Finish, Modal issue fixes require. Keep the Welcome Modal component aside.